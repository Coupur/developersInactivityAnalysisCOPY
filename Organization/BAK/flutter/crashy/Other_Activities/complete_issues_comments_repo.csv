id,date,creator_login
330347688,2017-09-18 20:33:39,yjbanov
319172586,2017-07-31 19:33:57,eseidelGoogle
320782902,2017-08-07 21:15:46,cbracken
371986492,2018-03-10 00:51:22,Hixie
