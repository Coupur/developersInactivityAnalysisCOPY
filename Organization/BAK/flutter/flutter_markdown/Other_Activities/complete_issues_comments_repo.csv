id,date,creator_login
341660827,2017-11-03 09:58:00,mit-mit
341405343,2017-11-02 12:25:37,mit-mit
341401444,2017-11-02 12:07:05,mit-mit
341401009,2017-11-02 12:04:52,mit-mit
327211208,2017-09-05 15:25:26,eseidelGoogle
333241521,2017-09-29 21:18:22,Hixie
333256145,2017-09-29 22:45:14,Hixie
341692498,2017-11-03 12:40:47,mit-mit
341689180,2017-11-03 12:23:54,mit-mit
341403286,2017-11-02 12:15:56,mit-mit
342092064,2017-11-06 09:28:18,mit-mit
341823826,2017-11-03 20:52:00,mit-mit
342101409,2017-11-06 10:04:24,mit-mit
342090333,2017-11-06 09:21:34,mit-mit
342234255,2017-11-06 18:06:04,mit-mit
353467527,2017-12-21 21:56:25,Hixie
353290841,2017-12-21 08:37:28,mit-mit
355168701,2018-01-04 00:42:40,Hixie
366994905,2018-02-20 14:31:41,mit-mit
361493167,2018-01-30 06:47:17,Hixie
374099506,2018-03-19 04:39:12,Hixie
374099241,2018-03-19 04:36:24,Hixie
366068131,2018-02-15 21:31:04,Hixie
377066000,2018-03-28 23:01:22,jonahwilliams
377065417,2018-03-28 22:58:16,cbracken
377052404,2018-03-28 22:02:20,Hixie
377063792,2018-03-28 22:50:52,cbracken
377064249,2018-03-28 22:53:05,cbracken
399921129,2018-06-25 11:34:28,mit-mit
399515814,2018-06-22 17:16:41,xster
392974003,2018-05-29 22:56:54,Hixie
392906414,2018-05-29 19:14:05,Hixie
392906173,2018-05-29 19:13:26,Hixie
384472300,2018-04-26 00:13:15,Hixie
384472268,2018-04-26 00:13:04,Hixie
384457970,2018-04-25 22:49:06,Hixie
380595363,2018-04-11 21:05:04,Hixie
379983362,2018-04-10 05:59:53,Hixie
379971367,2018-04-10 04:38:08,Hixie
399919760,2018-06-25 11:28:20,mit-mit
399918520,2018-06-25 11:22:44,mit-mit
409651373,2018-08-01 17:15:27,zanderso
449657317,2018-12-23 19:04:39,Hixie
426417083,2018-10-02 20:24:32,Hixie
424538113,2018-09-25 23:47:35,Hixie
412606019,2018-08-13 17:51:36,Hixie
412541346,2018-08-13 14:39:13,mit-mit
412259754,2018-08-11 08:12:55,mit-mit
418551904,2018-09-04 23:39:46,Hixie
414648807,2018-08-21 11:57:24,mit-mit
414305282,2018-08-20 12:50:30,mit-mit
414654718,2018-08-21 12:21:19,mit-mit
430440889,2018-10-16 23:52:56,Hixie
424536499,2018-09-25 23:38:02,Hixie
424536487,2018-09-25 23:37:58,Hixie
499714790,2019-06-07 00:16:14,GaryQian
499708582,2019-06-06 23:46:25,GaryQian
499678207,2019-06-06 21:40:34,GaryQian
499660155,2019-06-06 20:43:34,GaryQian
499189283,2019-06-05 17:55:24,GaryQian
