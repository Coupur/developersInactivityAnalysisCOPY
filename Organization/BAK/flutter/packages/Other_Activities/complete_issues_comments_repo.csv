id,date,creator_login
398406910,2018-06-19 13:53:39,mit-mit
398240294,2018-06-19 00:54:36,gspencergoog
397916037,2018-06-18 00:12:47,Hixie
397724006,2018-06-15 19:45:15,gspencergoog
397723694,2018-06-15 19:43:48,gspencergoog
397722316,2018-06-15 19:37:21,Hixie
408142111,2018-07-26 15:44:49,gspencergoog
408133153,2018-07-26 15:18:49,gspencergoog
403121484,2018-07-06 19:06:39,gspencergoog
401955905,2018-07-02 22:23:51,Hixie
398278847,2018-06-19 05:30:00,gspencergoog
415544884,2018-08-23 19:36:53,gspencergoog
414541415,2018-08-21 03:52:18,gspencergoog
427473054,2018-10-05 19:22:40,gspencergoog
457355340,2019-01-24 21:02:58,dnfield
457353549,2019-01-24 20:57:16,gspencergoog
456695971,2019-01-23 07:17:05,dnfield
453664253,2019-01-11 21:35:14,dnfield
436521475,2018-11-07 06:46:00,dnfield
432086056,2018-10-23 04:22:34,Hixie
431909053,2018-10-22 17:36:02,dnfield
431890622,2018-10-22 16:38:11,Hixie
431887125,2018-10-22 16:27:16,Hixie
431523720,2018-10-19 23:09:03,gspencergoog
431521815,2018-10-19 22:57:04,Hixie
431209707,2018-10-19 00:49:12,dnfield
431190297,2018-10-18 22:52:27,dnfield
431165998,2018-10-18 21:14:52,dnfield
433577266,2018-10-27 00:47:40,yjbanov
434770808,2018-10-31 17:11:05,gspencergoog
456680030,2019-01-23 05:55:50,dnfield
446527113,2018-12-12 09:50:22,vlidholt
457375983,2019-01-24 22:06:37,gspencergoog
488177968,2019-05-01 02:27:44,dnfield
488176473,2019-05-01 02:15:54,dnfield
489238323,2019-05-03 21:00:19,dnfield
513284918,2019-07-19 16:04:54,gspencergoog
513104233,2019-07-19 06:08:59,tvolkert
513093522,2019-07-19 05:16:08,tvolkert
513092038,2019-07-19 05:07:30,tvolkert
513090265,2019-07-19 04:56:35,tvolkert
