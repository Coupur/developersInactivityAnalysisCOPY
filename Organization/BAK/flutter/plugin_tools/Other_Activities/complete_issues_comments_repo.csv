id,date,creator_login
301608314,2017-05-15 21:20:23,tvolkert
301537853,2017-05-15 17:02:45,collinjackson
303817098,2017-05-24 18:49:55,goderbauer
303813794,2017-05-24 18:37:34,devoncarew
303811994,2017-05-24 18:31:07,devoncarew
306557077,2017-06-06 17:21:38,mehmetf
306553233,2017-06-06 17:08:17,goderbauer
306375862,2017-06-06 04:04:04,mehmetf
308477427,2017-06-14 15:59:24,mehmetf
363285848,2018-02-06 02:02:58,goderbauer
363284313,2018-02-06 01:54:09,goderbauer
380936597,2018-04-12 20:36:47,mravn-google
380936375,2018-04-12 20:35:56,goderbauer
380934788,2018-04-12 20:30:10,mravn-google
397353420,2018-06-14 16:18:18,mit-mit
397746616,2018-06-15 21:26:58,mravn-google
398129663,2018-06-18 17:18:30,goderbauer
397831292,2018-06-16 18:36:56,mravn-google
398129535,2018-06-18 17:18:09,goderbauer
397831699,2018-06-16 18:44:10,mravn-google
405364330,2018-07-16 20:03:56,gspencergoog
431189562,2018-10-18 22:48:34,dnfield
431189340,2018-10-18 22:47:27,dnfield
431193152,2018-10-18 23:07:09,gspencergoog
431202571,2018-10-19 00:01:25,gspencergoog
431470960,2018-10-19 19:21:06,gspencergoog
513485952,2019-07-20 17:37:37,collinjackson
