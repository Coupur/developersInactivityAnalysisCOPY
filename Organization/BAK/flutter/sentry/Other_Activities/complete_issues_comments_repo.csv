id,date,creator_login
320783763,2017-08-07 21:19:17,cbracken
319719558,2017-08-02 16:07:17,yjbanov
319676127,2017-08-02 13:44:10,eseidelGoogle
322341771,2017-08-15 00:00:09,yjbanov
322340378,2017-08-14 23:50:18,xster
322343470,2017-08-15 00:12:37,xster
330314388,2017-09-18 18:28:51,cbracken
344671826,2017-11-15 17:44:51,yjbanov
344662510,2017-11-15 17:14:38,Hixie
369672715,2018-03-01 17:47:29,yjbanov
369061132,2018-02-27 23:12:25,gspencergoog
374823648,2018-03-21 03:16:09,yjbanov
374769016,2018-03-20 21:49:25,yjbanov
374768756,2018-03-20 21:48:33,yjbanov
374767525,2018-03-20 21:44:13,yjbanov
374764440,2018-03-20 21:33:22,yjbanov
374750568,2018-03-20 20:46:10,Hixie
374764943,2018-03-20 21:35:11,xster
374768015,2018-03-20 21:46:07,xster
375053917,2018-03-21 18:43:16,xster
384719652,2018-04-26 17:15:57,yjbanov
384715785,2018-04-26 17:03:05,yjbanov
384715230,2018-04-26 17:01:17,yjbanov
384706558,2018-04-26 16:33:02,yjbanov
384705785,2018-04-26 16:30:42,yjbanov
384703121,2018-04-26 16:22:32,yjbanov
381684415,2018-04-16 17:26:04,yjbanov
389246123,2018-05-15 17:21:10,yjbanov
389240054,2018-05-15 17:00:56,yjbanov
389027734,2018-05-15 03:01:12,yjbanov
389027445,2018-05-15 02:59:15,yjbanov
389253381,2018-05-15 17:44:18,yjbanov
404048486,2018-07-11 05:21:48,yjbanov
403549076,2018-07-09 17:02:57,yjbanov
403545418,2018-07-09 16:50:51,tvolkert
403540378,2018-07-09 16:33:45,yjbanov
501732450,2019-06-13 14:40:27,yjbanov
434513696,2018-10-30 23:58:15,Hixie
420490052,2018-09-12 02:23:20,eseidelGoogle
416786023,2018-08-29 00:45:04,yjbanov
416773206,2018-08-28 23:29:15,Hixie
409623121,2018-08-01 15:53:05,yjbanov
428254552,2018-10-09 16:12:38,yjbanov
428249571,2018-10-09 15:58:49,yjbanov
428010381,2018-10-08 23:25:17,yjbanov
427959289,2018-10-08 19:54:50,yjbanov
428248456,2018-10-09 15:55:52,yjbanov
428248430,2018-10-09 15:55:48,yjbanov
431200646,2018-10-18 23:49:33,yjbanov
445673963,2018-12-10 04:26:34,yjbanov
437543352,2018-11-10 01:10:58,yjbanov
437506122,2018-11-09 21:51:00,yjbanov
437216780,2018-11-09 01:23:34,yjbanov
433577402,2018-10-27 00:49:03,yjbanov
437505300,2018-11-09 21:47:36,yjbanov
445897894,2018-12-10 17:21:15,yjbanov
