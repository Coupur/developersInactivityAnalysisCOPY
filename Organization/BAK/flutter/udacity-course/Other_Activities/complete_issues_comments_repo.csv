id,date,creator_login
330942815,2017-09-20 18:38:04,sethladd
330705161,2017-09-19 23:49:16,sethladd
330651802,2017-09-19 19:42:50,devoncarew
330647894,2017-09-19 19:27:24,sethladd
330647755,2017-09-19 19:26:56,sethladd
330647612,2017-09-19 19:26:24,sethladd
332637533,2017-09-27 19:57:18,collinjackson
332636650,2017-09-27 19:54:00,collinjackson
334888346,2017-10-06 22:53:32,sethladd
334642417,2017-10-06 02:28:36,collinjackson
334585129,2017-10-05 20:38:25,sethladd
334530030,2017-10-05 17:08:24,sethladd
333612252,2017-10-02 17:53:06,collinjackson
334584812,2017-10-05 20:37:19,sethladd
344452431,2017-11-15 01:05:17,yjbanov
344452179,2017-11-15 01:03:48,yjbanov
335278888,2017-10-09 20:39:39,sethladd
344669171,2017-11-15 17:35:54,Hixie
340114115,2017-10-27 22:42:13,xster
353203676,2017-12-20 22:39:38,Hixie
352123410,2017-12-15 22:03:50,yjbanov
349365401,2017-12-05 16:48:04,collinjackson
344717130,2017-11-15 20:24:08,collinjackson
345096639,2017-11-16 23:29:07,yjbanov
346423053,2017-11-22 17:37:36,yjbanov
345595895,2017-11-20 05:33:01,yjbanov
354864397,2018-01-02 20:14:24,yjbanov
355441125,2018-01-05 00:39:18,yjbanov
360936851,2018-01-26 23:50:59,yjbanov
360936825,2018-01-26 23:50:47,yjbanov
361353394,2018-01-29 19:12:46,yjbanov
364183671,2018-02-08 17:18:23,yjbanov
364184607,2018-02-08 17:21:26,yjbanov
364185714,2018-02-08 17:25:02,yjbanov
365021117,2018-02-12 18:42:47,yjbanov
365021392,2018-02-12 18:43:34,yjbanov
365024702,2018-02-12 18:54:40,yjbanov
365098661,2018-02-12 23:27:26,yjbanov
365676854,2018-02-14 17:09:37,yjbanov
369675377,2018-03-01 17:55:53,yjbanov
371958409,2018-03-09 22:05:20,yjbanov
371958629,2018-03-09 22:06:10,yjbanov
371958981,2018-03-09 22:07:36,yjbanov
372379879,2018-03-12 16:47:58,yjbanov
373154550,2018-03-14 19:57:15,yjbanov
373210291,2018-03-14 23:36:28,yjbanov
373144425,2018-03-14 19:22:08,jonahwilliams
373151482,2018-03-14 19:46:45,yjbanov
373155128,2018-03-14 19:59:24,yjbanov
373179911,2018-03-14 21:24:44,yjbanov
374405645,2018-03-19 22:30:08,yjbanov
377852943,2018-04-02 04:39:22,yjbanov
379900790,2018-04-09 21:32:42,Hixie
377856022,2018-04-02 05:09:39,yjbanov
377973360,2018-04-02 16:41:01,yjbanov
378750321,2018-04-04 21:25:17,yjbanov
378835317,2018-04-05 06:39:40,yjbanov
378828860,2018-04-05 06:01:00,yjbanov
379104147,2018-04-05 23:16:01,yjbanov
379144458,2018-04-06 04:24:30,yjbanov
379874059,2018-04-09 19:54:50,yjbanov
379875932,2018-04-09 20:01:22,yjbanov
379916707,2018-04-09 22:44:18,yjbanov
380202614,2018-04-10 18:31:01,yjbanov
380205093,2018-04-10 18:39:12,yjbanov
384443202,2018-04-25 21:40:24,Hixie
384121899,2018-04-25 00:26:49,Hixie
383182922,2018-04-20 18:27:29,Hixie
383170747,2018-04-20 17:42:34,Hixie
385093753,2018-04-27 21:04:20,yjbanov
392972173,2018-05-29 22:47:06,Hixie
400499354,2018-06-26 23:57:01,Hixie
400499423,2018-06-26 23:57:25,Hixie
401924219,2018-07-02 20:27:56,Hixie
481459594,2019-04-09 22:12:04,Hixie
401977376,2018-07-03 00:33:38,Hixie
415834957,2018-08-24 17:56:18,Hixie
