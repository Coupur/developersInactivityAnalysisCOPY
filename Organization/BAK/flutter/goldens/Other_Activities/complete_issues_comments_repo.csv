id,date,creator_login
402199969,2018-07-03 15:35:41,goderbauer
403676200,2018-07-10 02:08:01,aam
403948400,2018-07-10 20:04:09,cbracken
410101196,2018-08-02 23:44:10,gspencergoog
452883798,2019-01-09 21:57:11,dnfield
452883584,2019-01-09 21:56:30,dnfield
452883378,2019-01-09 21:55:55,dnfield
452877249,2019-01-09 21:37:08,dnfield
452857227,2019-01-09 21:12:53,dnfield
481890501,2019-04-10 22:31:08,xster
481774335,2019-04-10 16:57:26,dnfield
486423652,2019-04-24 20:59:14,dnfield
494538992,2019-05-21 20:08:22,justinmc
494535872,2019-05-21 19:59:19,GaryQian
512860939,2019-07-18 15:17:19,justinmc
